src/algorithms.jl

```diff
@@ -1,5 +1,5 @@
-const DEFAULT_NLSOLVE_SHOOTING = TrustRegion(; autodiff = Val(true))
-const DEFAULT_NLSOLVE_MIRK = NewtonRaphson(; autodiff = Val(true))
+const DEFAULT_NLSOLVE_SHOOTING = NewtonRaphson()
+const DEFAULT_NLSOLVE_MIRK = NewtonRaphson()
```

Comment:
Not so sure about that.

Reply:
This change to NewtonRaphson was before the TR bugs were fixed. We can revert this part. The `autodiff = true` is redundant wince that is the current default.